src/encoded/tests/test_types_biosample.py

```diff
@@ -161,3 +161,15 @@ def test_biosample_summary_construct(testapp,
     assert res.json['object']['summary'] == (
         'Drosophila melanogaster '
         'female (10 days) liver tissue stably expressing C-terminal eGFP-tagged ATF4 under daf-2 promoter')
+
+
+def test_perturbed(
+    testapp,
+    biosample_1,
+    interference_genetic_modification,
+):
+    testapp.patch_json(biosample_1['@id'], {
+        'genetic_modifications': [interference_genetic_modification['@id']],
+        })
+    res = testapp.get(biosample_1['@id']+'@@index-data')
+    assert res.json['object']['perturbed'] is True
```

Comment:
Maybe add a test for False before patching, @jenjou

Reply:
Yes, the more tests the better.

Reply:
Added a second test for not perturbed